core/src/main/java/com/datastax/oss/driver/internal/core/tracker/NoopRequestTracker.java

```diff
@@ -51,7 +53,8 @@ public void onError(
       long latencyNanos,
       @NonNull DriverExecutionProfile executionProfile,
       Node node,
-      @NonNull String requestPrefix) {
+      @NonNull String requestPrefix,
+      @NonNull ExecutionInfo executionInfo) {
```

Comment:
Similar to with `MultiplexingRequestTracker`, this is not marked `@NonNull` in the interface